src/client/kinto_http.rs

```diff
@@ -79,9 +78,23 @@ pub fn get_records(
             name: "no content-length header error".to_owned(),
         });
     debug!("Download {:?} bytes...", size);
-    let result: ChangesetResponse = resp.json()?;
 
-    Ok(result)
+    let latest_change: KintoPluralResponse<LatestChangeResponse> = resp.json()?;
+    
+    let last_modified = match latest_change.data.get(0) {
+        Some(change) => change.last_modified,
+        None => {
+            // bucket/collection provided is unknown
+            return Err(KintoError::Error {name: format!("404 - unknown bucket/collection name provided: bucket_name={}, collection_name{}", bid, cid)});
```

Comment:
I don't think 404 makes sense here. Error can just be `format!("Unknown collection {}/{}", bid, cid)`

Reply:
done